install-and-configure/advanced-configuration/resource-consumption.md

```diff
@@ -81,6 +81,8 @@ kubecostAggregator:
   dbConcurrentIngestionCount: 1
```

Comment:
We are recommending using `dbReadThreads: 1` above. I don't think it's necessary to change this recommendation but open to discussion

Reply:
There's some text above "By default these should all be set to `1`". We should delete this since it's no longer true?

Reply:
You're right, thanks for calling that out. The other two values still have those defaults, though. 
WDYT if we rephrase from 
"**Lower concurrency.** By default these should all be set to `1`."
to
"Settings for lower concurrency:" 
or something like that?

Reply:
What about this? To remain consistent with how it's worded in all the other paragraphs above?

```
**Lower concurrency.** Set values to `1`.
```

Reply:
Applied, thank you!